app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) == 3:
+        comm, old_file, new_file = command.split()
```

Comment:
Don't call split twice. You should create variable for storing result of split and using where you need